kolibri/core/discovery/utils/network/client.py

```diff
@@ -55,7 +55,7 @@ def _attempt_connections(self, urls):
                     info = response.json()
                     self.info = {}
                     for key in device_info_keys.get(DEVICE_INFO_VERSION, []):
-                        self.info[key] = info.get(key)
+                        self.info[key] = info.get(key) or False
```

Comment:
If we move the `devince_info_defaults` into this file, we can then do:
```
self.info[key] = info.get(key, device_info_defaults.get(key))
```

Reply:
Done